netbout-web/src/main/java/com/netbout/rest/account/TkIndex.java

```diff
@@ -62,7 +63,7 @@ public Response act(final Request req) throws IOException {
         return new RsPage(
             "/xsl/account.xsl",
             this.base,
-            req,
+            new RqWithDefaultHeader(req, "accept", "text/xml"),
```

Comment:
@dmzaytsev what do you think of other comments? I tend to disagree with @pinaf in other things